src/refract/RenderJSONVisitor.cc

```diff
@@ -143,24 +118,19 @@ namespace refract
 
     void RenderJSONVisitor::operator()(const EnumElement& e) {
 
-        if (!enumValue) { // there is no enumValue injected from ExtendElement,try to pick value directly
-
-            const EnumElement::ValueType* val = GetEnumValue(e);
-            if (val && !val->empty()) {
-                enumValue = val->front()->clone();
-            }
-            else {
-                enumValue = new StringElement;
-            }
+        const IElement* val = GetValue<EnumElement>(e);
+        if (val && !val->empty()) {
+            val = val->clone();
+        }
+        else {
+            val = new StringElement;
```

Comment:
Wondering why we default to a string element? Shouldn't there be no value if I have an empty enum (i.e):

```apib
+ myThing (enum)
```